File: scripts/c/basicmath.c
#include<stdio.h>
int main()
{
	int a,b,c;
	printf("Enter the a value \n");
	scanf("%d",&a);
	printf("Enter the b value \n");
	scanf("%d",&b);
	c=a+b;
	printf("The Addition of two numbers is %d \n",c);
	c=a-b;
	printf("The Subtraction of two numbers is %d \n",c );
	c=a*b;
	printf("The Multiplication of two numbers is %d\n",c);
	c=a/b;
	printf("The Division of two numbers is %d \n",c);
	c=a%b;
	printf("The Muduless of two number is %d \n",c);
}

File: scripts/java/BasicMath.java
import java.io.*;
class BasicMath
{
	public static void main(String args[])throws IOException
	{
		int a,b,c;
		DataInputStream dis=new DataInputStream(System.in);
		System.out.println("Enter the a value");
		a=Integer.parseInt(dis.readLine());
		System.out.println("Enter the b value");
		b=Integer.parseInt(dis.readLine());
		c=a+b;
		System.out.println("The Addition of two values is "+c);
		c=a-b;
		System.out.println("The Subtraction of two values is "+c);
		c=a*b;
		System.out.println("The Multiplication of two values is "+c);
		c=a/b;
		System.out.println("The Division of two values is "+c);
		c=a%b;
		System.out.println("The Moduless of two values is "+c);
	}
}